packages/renderer/src/lib/kube/pods/terminal/KubernetesTerminalService.spec.ts

```
@@ -18,23 +18,24 @@
+import type { PodContainerInfo } from '@podman-desktop/api';
 import { beforeEach, expect, test } from 'vitest';
 
 import { TerminalService } from '/@/lib/kube/pods/terminal/KubernetesTerminalService';
+import type { PodInfoUI } from '/@/lib/pod/PodInfoUI';
 
 let terminalService: TestableKubernetesTerminalService;
 
 class TestableKubernetesTerminalService extends TerminalService {
-  public testInvalidateCacheRecordOnStatusUpdate(podsInfos: PodInfo[]): void {
+  public testInvalidateCacheRecordOnStatusUpdate(podsInfos: PodInfoUI[]): void {
     return this.invalidateCacheRecordOnStatusUpdate(podsInfos);
```

Biome (2.5.3)

[error] The function should not return a value because its return type is void.

(lint/correctness/noVoidTypeReturn)